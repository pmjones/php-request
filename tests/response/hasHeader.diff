001+ Fatal error: Uncaught Error: Call to undefined method SapiResponse::hasHeader() in /Users/pmjones/Code/pmjones/php-request/tests/response/hasHeader.php:4
002+ Stack trace:
003+ #0 {main}
001- bool(true)
002- bool(true)
003- bool(false)
004+   thrown in /Users/pmjones/Code/pmjones/php-request/tests/response/hasHeader.php on line 4