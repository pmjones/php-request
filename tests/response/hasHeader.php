<?php
$response = new SapiResponse();
$response->setHeader('Foo-Bar', 'baz');
var_dump($response->hasHeader('Foo-Bar'));
var_dump($response->hasHeader('foo-bar'));
var_dump($response->hasHeader('dib-zim'));
