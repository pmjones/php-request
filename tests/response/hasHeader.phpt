--TEST--
SapiResponse::hasHeader
--FILE--
<?php
$response = new SapiResponse();
$response->setHeader('Foo-Bar', 'baz');
var_dump($response->hasHeader('Foo-Bar'));
var_dump($response->hasHeader('foo-bar'));
var_dump($response->hasHeader('dib-zim'));
--EXPECT--
bool(true)
bool(true)
bool(false)
