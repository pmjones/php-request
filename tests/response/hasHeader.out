Fatal error: Uncaught Error: Call to undefined method SapiResponse::hasHeader() in /Users/pmjones/Code/pmjones/php-request/tests/response/hasHeader.php:4
Stack trace:
#0 {main}
  thrown in /Users/pmjones/Code/pmjones/php-request/tests/response/hasHeader.php on line 4